types/tx2/service.pb.go

```diff
@@ -8,7 +8,7 @@ import (
 	fmt "fmt"
 	query "github.com/Finschia/finschia-sdk/types/query"
 	tx "github.com/Finschia/finschia-sdk/types/tx"
-	types1 "github.com/Finschia/ostracon/proto/ostracon/types"
+	types1 "github.com/tendermint/tendermint/proto/tendermint/types"
```

Comment:
This stub file is changed without changing proto file.

Reply:
DITTO